jamjar/jamjar/common/video_utils.py

```diff
@@ -0,0 +1,56 @@
+from django.conf import settings
+
+import uuid
+import os
+
+import logging
+
+from jamjar.tasks.transcode_video import transcode_video
+
+class VideoUtils(object):
+    BASE_PATH = settings.VIDEOS_PATH
```

Comment:
@markkohdev did you one better -- I moved ALL of them to the video model and added unit tests :)